src/Uno.UI.RuntimeTests/Tests/Windows_ApplicationModel/DataTransfer/Given_Clipboard.cs

```diff
@@ -0,0 +1,51 @@
+#if __ANDROID__ || __MACOS__
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Resources;
+using Windows.ApplicationModel.Resources.Core;
+
+// Testing Append, Write and Read in one test method
+
+namespace Uno.UI.RuntimeTests.Tests
+{
+	[TestClass]
+	public class Given_Clipboard
+	{
+
+		[TestInitialize]
+		public void Init()
+		{
+		}
+
+		[TestCleanup]
+		public void Cleanup()
+		{
+		}

```

Comment:
Please remove when not used.

Reply:
If those are empty, you can remove them.